project-base/app/src/DataFixtures/Demo/ComplaintDataFixture.php

```diff
@@ -50,9 +54,14 @@ public function load(ObjectManager $manager): void
         $customerUser1 = $this->getReference(CustomerUserDataFixture::CUSTOMER_PREFIX . 1);
         /** @var \App\Model\Order\Order $order1 */
         $order1 = $this->getReference(OrderDataFixture::ORDER_PREFIX . 1);
+        $uploadedFile1 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/400.jpg');
+        $uploadedFile2 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/401.jpg');
+        $uploadedFile3 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/402.jpg');
+        $uploadedFile4 = $this->createUploadedFiles(__DIR__ . '/../resources/images/complaint/403.jpg');
+
```

Comment:
All emages in data fixtures are handled in ImageDataFixture.php and I feel that this should be done same way. Can you review it please?

Reply:
What license is there for those images? We cannot use just some images, they need to have license free to use for commercial use.